lntest/itest/lnd_channel-funding_fund_max_test.go

```diff
@@ -0,0 +1,324 @@
+// +build rpctest
+
+package itest
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/btcsuite/btcutil"
```

Comment:
Erroneous import:

```suggestion
	"github.com/btcsuite/btcd/btcutil"
```

Also I'm no go expert but other test files contain a `+build xxx` in the top:

```diff
+//go:build !rpctest
// +build !rpctest
```

Reply:
Fixed in the continuation of this PR, here: https://github.com/lightningnetwork/lnd/pull/6903